documentation/API-authentication-token.md

```diff
@@ -1,12 +1,19 @@
 # API authentication token
 
-< to be described >
+When requesting a token on the oauth2 token resource using the client_credentials a queryparameter containing the identifier for the school. A scope to access data can optionally be requested and if consent has been given, will be added as a scope in the token defined below.
 
-jwt token with as minimum the following attributes:
+Signed JWT with as minimum the following attributes:
 
-* scopes
-* party identity
-* school identity (optional)
+client_id : identifier of application that has request this token (recipient)
+scopes : list of scopes that client_id wants to access
+
+optional (proposal):
```

Comment:
Jorim, I would remove these from the token definition and explain that the idp can do this anyway they like as long as they can access it when the token is used in a API Call.

Reply:
This gives each idp the freedom to do it in their own way and not have too many dependencies